.github/workflows/build.yml

```diff
@@ -38,6 +38,21 @@ jobs:
           fi
           echo "alias_tags=${alias_tags[*]}" >> $GITHUB_OUTPUT
 
+      - name: Install just
+        id: install_just
+        uses: extractions/setup-just@v1
+
+      - name: Check just syntax
+        id: check_just_syntax
+        shell: bash
+        run: |
+          for file in build/ublue-os-just/*; do
```

Comment:
Should be *.just

Reply:
Oh yeah sorry, totally missed that! Will fix.